docs/source/changes.md

```diff
@@ -18,6 +18,8 @@ authors:
 
 ### Behavior changes
 
+- The [`conditions`][config.conditions] setting will now be `None` by default. 
+  It is a required setting so it will raise an error if left as `None`.
```

Comment:
Something like this?

```suggestion
  It is a required setting so it will raise an error if left as `None`.
  ({{ gh(348) }} by {{ authors.guiomar, authors.hoechenberger }})
```

Reply:
No need to include me here :) I'm just reviewing
But yes, otherwise exactly like this!!

Reply:
I think we did it half, thanks for showing me :)